docs/earthfile/earthfile.md

```diff
@@ -153,7 +153,7 @@ This option is deprecated. Use `--<build-arg-key>=<build-arg-value>` instead.
 
 #### Synopsis
 
-* `RUN [--push] [--entrypoint] [--privileged] [--secret <env-var>=<secret-id>] [--ssh] [--mount <mount-spec>] [--] <command>` (shell form)
+* `RUN [--push] [--entrypoint] [--privileged] [--secret <env-var>=<secret-id>] [--network=none] [--ssh] [--mount <mount-spec>] [--] <command>` (shell form)
 * `RUN [[<flags>...], "<executable>", "<arg1>", "<arg2>", ...]` (exec form)
```

Comment:
We used this in other places where listing all the flags was too much. Perhaps it's time for this one too.

```suggestion
* `RUN [options...] [--] <command>` (shell form)
* `RUN [[options...], "<executable>", "<arg1>", "<arg2>", ...]` (exec form)
```